O3-1.tex

```diff
@@ -319,8 +317,8 @@ \section{The LSST software stack}
 Public License, Version 3. The stack
 prototype code and documentation are available via \url{http://ls.st/ug}.
 Its open source nature, an open development
-process, attention to software engineering, a long-term project
-commitment and a modular design that can be modified for use with
+process, a long-term project
```

Comment:
@ktlim Okay so let's take a moment back and consider this is a paper written for the proceedings of an astronomical software conference. 

As to your first comment, i.e. "We're at least trying to do some software engineering, which I am led to believe is unlike many other projects.". Failing an actual way of differentiating between projects that *tried* to do software engineering and failed, and projects that did not try ('Do or do not, there is no try') at the very least I'd say, Wikipedian style, "citation needed". How could you measure that? But scientific objectivity aside (as I said, it is a paper after all) , what is the *point* of such a statement? The audience for this paper are astronomical software engineers. This statement will come across as arrogant and patronising even to anybody who *might* be a deserving target, never mind to those who don't deserve it. There is zero purpose in asserting this. If it is true, it is self-evident, and if it is not, it should not be asserted. 

As to the modularity: I do not wish to engage in a legalistic argument as to what the term modular means in computer science, but if we're into lawyering this I will say that the relevant legal standard is the "resasonable person". Would a reasonable software engineer think that the current implementation is characterised by its modular design? You may protest that I am not a reasonable person, but I at least, do not think so. Again, what purpose is served by making such an assertion? 

There's no glory in winning an argument against a straw man.  Call no man happy before he is dead, said Solon, and boast not about your software until it is in production, say I.